### ISSUE TITLE
Python import typo

### ISSUE THREAD

**Issue by nitrotm** [pylakey/aiotdlib#9]
It seems this should be written `from .api import` in client.py (missing dot):

https://github.com/pylakey/aiotdlib/blob/cd877506bb9e90d15bdee5b8c89af1f73cdcf8df/aiotdlib/client.py#L30

Because it fails when importing the package when installed with pip:

```
  File "/home/user/.local/lib/python3.9/site-packages/aiotdlib/client.py", line 30, in <module>
    from api import (
ModuleNotFoundError: No module named 'api'
```

**Comment by hwmbing11** [comment]
sys.path.append('/home/user/.local/lib/python3.9/site-packages/aiotdlib/client.py')

Call before "import aiotdlib"

**Comment by pylakey** [comment]
Fixed in 0.15.1
